### DISCUSSION TITLE
Harpoon's quick menu

### DISCUSSION THREAD

**Discussion by cschmatzler** [folke/edgy.nvim#42] [Q&A]
<img width="1840" alt="image" src="https://github.com/folke/edgy.nvim/assets/8415974/64139895-97db-49c0-b9d8-14ae90264902">

Toggling Harpoon's quick menu with `:lua require("harpoon.ui").toggle_quick_menu()` with edgy set up with `{ ft = "harpoon",         title = "Harpoons", size = { width = 0.3 } }` results in the actual list of harpoon rendered in the edgy panel, but the normal harpoon popup still appears, albeit empty.
Is there any way to stop that popup from showing up?

Cheers

**Comment by folke** [comment]
probably because that popup is rendered as two windows over each other and the other window likely has a different ft.
